src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1165,8 +1169,12 @@
     <value>Add new</value>
     <comment>Button label that creates a new color scheme.</comment>
   </data>
-  <data name="ColorScheme_DeleteButton.Text" xml:space="preserve">
-    <value>Delete color scheme</value>

```

Comment:
I'll fix this after lunch. We crash because the button doesn't have text. This used to work because we had icons on all of these buttons, so the resource would bind to the text block inside the button. Now, these buttons don't have icons.

We should probably add icons to all of these. I'll do that in this next commit.